lang/fr.xml

```diff
@@ -423,6 +419,14 @@ Votre système Vita3K est prêt !</completed_setup>
 		</language>
 	</settings>
 
+	<settings_dialog>
+		<close>Close</close>
+		<save_reboot>Save & Reboot</save_reboot>
+		<save_apply>Save & Apply</save_apply>
+		<save>Sauvegarder</save>
+		<keep_changes>Click on Save to keep your changes.</keep_changes>
+	</settings_dialog>
+
```

Comment:
no set lang string if you keep it in english, just using global variable
same for all